app/models/event.rb

```diff
@@ -1,4 +1,9 @@
 class Event < ApplicationRecord
   # associations
-  belongs_to :user
+  belongs_to :creator, class_name: 'User', foreign_key: 'user_id'
+  has_many :event_attendances
```

Comment:
- [ ] I can't find a way for a specific user to attend an event -- Not yet implemented 

We can start by adding a new association to events model using `has_many :through` to associate the event with attendances like so:
```ruby
has_many :attendees, through: :event_attendances
```
And to make a user ie `user` attend an event ie `event` you can simpley `event << user`

Reply:
I am not sure this was one of the requirements stated in the odin project.

Reply:
Hi @igakigongo 
This is from Odin specifications 


![Screenshot from 2019-10-23 13-01-06](https://user-images.githubusercontent.com/4246637/67386481-59fa0300-f595-11e9-8826-8ce38ce9dcb1.png)

I couldn't find `A user can attend many events. An event can be attended by many users` feature in your application. The above code is a suggestion on how you start implementing this feature, but surely you have the freedom of implementing it the way you like.